[February 7 2024] Adding Valentin Delaye's spotlight (#77)
jenkins-infra/contributor-spotlight: src/pages/contributors/valentin-delaye.adoc

= Valentin Delaye
:page-name: Valentin Delaye
:page-linkedin: 
:page-twitter: 
:page-github: jonesbusy
:page-email: 
:page-image: avatar/valentin-delaye.png
:page-pronouns: He/Him/His
:page-location: Lausanne, Switzerland

> :page-firstcommit: 2009

Reviewer: User since 2009 but first commit in 2018

Reviewer 2: Out of scope for this PR but having both values could be nice.

:page-datepublished: 2024-02-07
:page-featured: true

> :page-intro: Valentin is a Jenkins contributor and plugin maintainer who's passionate about technology and programming. This passion is so great that it has spread as both his profession and hobby. He believes strongly in the power of open source and has been a Linux user since his teenage years. Valentin believes that open source is not only about consuming and taking advantage of it, but also contributing to it.

Reviewer 2: ```suggestion
:page-intro: Valentin is a Jenkins contributor and plugin maintainer who is passionate about technology and programming. This passion is so great that it has spread as both his profession and hobby. He believes strongly in the power of open source and has been a Linux user since his teenage years. Valentin believes that open source is not only about consuming and taking advantage of it, but also contributing to it.
```
Nit, I might be wrong but this abbreviation here sounds a bit strange to me.

Author: I definitely get what you mean here @lemeurherve, I'm okay with changing it accordingly. It still makes sense and is the same message, so there's no difference and makes it easier for others.

> :page-intro: Valentin is a Jenkins contributor and plugin maintainer who is passionate about technology and programming. This passion is so great that it has spread as both his profession and hobby. He believes strongly in the power of open source and has been a Linux user since his teenage years. Valentin believes that open source is not only about consuming and taking advantage of it, but also contributing to it.

Reviewer 3: ```suggestion
:page-intro: Valentin is a Jenkins contributor and plugin maintainer passionate about technology and programming. This passion is so great that it has spread as his profession and hobby. He believes strongly in the power of open source and has been a Linux user since his teenage years. Valentin believes that open source is about consuming, taking advantage of, and contributing to it.
```

== What is your background prior to contributing to Jenkins?

> Jenkins is the first open-source project I contributed to with a community.

Reviewer 2: ```suggestion
Jenkins is the first open-source project I contributed to with that kind of community.
```
Optional suggestion underlining the difference to other projects.

> Before that, I had submitted some small contributions on home-made Maven plugins.

Reviewer 3: ```suggestion
Before that, I had submitted some small contributions on homemade Maven plugins.
```

Now, I'm contributing to different tools and technologies like Helm charts (link:https://github.com/bitnami/charts/tree/main/bitnami/external-dns[Bitnami external DNS]) and Ansible collections (link:https://github.com/prometheus-community/ansible[Prometheus collection]).
Recently I've added some features to link:https://github.com/openrewrite/rewrite[OpenRewrite], which is also used by the Jenkins project.

== How long have you been using Jenkins?

I've been using Jenkins for quite a long time.

> I think the first time I used Jenkins was around 2009, before it was even called Jenkins.

Reviewer 3: ```suggestion
I think the first time I used Jenkins was around 2009 before it was even called Jenkins.
```

== Why choose Jenkins over other projects?

I gravitated towards Jenkins for its cost, flexibility, and ability to circumvent on-premises infrastructure constraints.
Since Jenkins is an open-source project, it can be easily extended via plugins.

== What problems has Jenkins solved for you?

Jenkins has helped me with automation and employing continuous integration despite on-premises environment limitations.

== Is there an aspect of Jenkins that you're particularly passionate about?

The community of course!
Jenkins also has a massive plugin ecosystem that provides further customization and abilities.

== What sort of contributions have felt the most successful or impactful?

> There isn't one contribution in particular, but over the last two years I've adopted about 10 plugins that were previously abandoned.

Reviewer: Now is 30 plugins

> There isn't one contribution in particular, but over the last two years I've adopted about 10 plugins that were previously abandoned.

Reviewer 3: ```suggestion
There isn't one contribution in particular, but over the last two years, I've adopted about 30 previously abandoned plugins.
```

> I've refreshed them, fixed any issues, including security, and I now maintain them regularly.

Reviewer 3: ```suggestion
I've refreshed them and fixed any issues, including security, and I now maintain them regularly.
```

Now, I'm contributing to different tools and technologies like Helm charts (link:https://github.com/bitnami/charts/tree/main/bitnami/external-dns[Bitnami external DNS]) and Ansible collections (link:https://github.com/prometheus-community/ansible[Prometheus collection]).
Recently I've added some features to link:https://github.com/openrewrite/rewrite[OpenRewrite], which is also used by the Jenkins project.

== How long have you been using Jenkins?

I've been using Jenkins for quite a long time.
I think the first time I used Jenkins was around 2009 before it was even called Jenkins.
My first contribution to Jenkins occurred in 2018.

== Why choose Jenkins over other projects?

I gravitated towards Jenkins for its cost, flexibility, and ability to circumvent on-premises infrastructure constraints.
Since Jenkins is an open-source project, it can be easily extended via plugins.

== What problems has Jenkins solved for you?

Jenkins has helped me with automation and employing continuous integration despite on-premises environment limitations.

== Is there an aspect of Jenkins that you're particularly passionate about?

The community of course!

> Jenkins also has a massive plugin ecosystem that provides further customization and abilities.

Reviewer: @kmartens27 Can you add a sentence at the end something like ?

"I'm also particularly interested by cloud-native deployment of Jenkins (like external storage) or Jenkinsfile runner"

Author: Sure thing!

== What sort of contributions have felt the most successful or impactful?

There isn't one contribution in particular, but over the last two years I've adopted about 30 previously abandoned plugins.

> I've refreshed them and fixed any issues, including security, and I now maintain them regularly.

Reviewer: "Over the last two years I've adopted about 30 previously abandoned plugins. I've refreshed them and fixed any issues, including security, and I now maintain them regularly"

Can you add a new sentence (feel free to rephrase it)

"But, if I need to choose the contribution I'm the most proud of is the recent addition of OpenCover and NUnit parser into the coverage plugin"

Author: Of course, thanks very much @jonesbusy !
